script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt

```diff
@@ -144,8 +144,8 @@ On the way, I see something I don't like.
 Even though I'm wearing my glasses, I can still see those 'lines'.
 <Page34>
 "Ugh......!"
-My eyeballs can't take the nausea that's been building up.
-It's a reaction to the fact that I've been trying not to see them for the last seven years.
+My eyeballs can't take the nausea that's building up.
+It's the consequence/reprecussion to the fact that I've been trying not to see them for the last seven years.
```

Comment:
Not a big fan of this change either. Ignoring the 'consequence to' syntax, I think reaction is more correct.